tailor_distro/create_recipes.py

```diff
@@ -23,6 +23,16 @@ def nested_update(d, u):
     return d
 
 
+def check_distro(recipe, os_version):
+    tmp = deepcopy(recipe)
+    for distro, config in recipe["distributions"].items():
+        os_list = config.get("os", [])
+        if len(os_list) != 0:
+            if os_version not in os_list:
+                del tmp["distributions"][distro]
```

Comment:
would it make sense to define distributions under the `common` key, instead of on a per-recipe basis?